### ISSUE TITLE
followingRedirects causes a TypeError

### ISSUE THREAD

**Issue by DeGraciaMathieu** [inspector-apm/inspector-laravel#17]
Hello,

**Describe the bug**

In a test suites, use a `followingRedirects` causes a `TypeError`.

```php
public function random()
{
    $clip = Clip::factory()->create();

    $response = $this->followingRedirects()->get('clips/random');

    $response
        ->assertStatus(200)
        ->assertSee($clip->title);
}
```

The controller behind the `clips/random` url simply performs a redirect.

```php
return Redirect::route('clips.show', $clip->slug)
```

Here is the full backtrace

```
1) Tests\Feature\ClipTest::random
TypeError: Argument 2 passed to Inspector\Laravel\Middleware\WebRequestMonitoring::terminate() must be an instance of Symfony\Component\HttpFoundation\Response, instance of Illuminate\Testing\TestResponse given, called in C:\laragon4\www\megasaurus\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php on line 208

C:\laragon4\www\megasaurus\vendor\inspector-apm\inspector-laravel\src\Middleware\WebRequestMonitoring.php:75
C:\laragon4\www\megasaurus\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php:208
C:\laragon4\www\megasaurus\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php:179
C:\laragon4\www\megasaurus\vendor\laravel\framework\src\Illuminate\Foundation\Testing\Concerns\MakesHttpRequests.php:515
C:\laragon4\www\megasaurus\vendor\laravel\framework\src\Illuminate\Foundation\Testing\Concerns\MakesHttpRequests.php:292
C:\laragon4\www\megasaurus\tests\Feature\ClipTest.php:46
```

**Expected behavior**

Nothing, the key env `INSPECTOR_ENABLE` is configured to `false` during testing.

**Web**
 - Chrome Binary
 - Package Version ^4.6
 - Laravel 8.10.0

Thank you !

**Comment by ilvalerione** [comment]
A new version was released (4.6.8) to fix the method signature conflict.

Thank you for your report.

**Comment by DeGraciaMathieu** [comment]
Thank you, happy New Year !
